api/ipfsproxy/ipfsproxy.go

```diff
@@ -692,6 +701,11 @@ func (proxy *Server) repoGCHandler(w http.ResponseWriter, r *http.Request) {
 			}
 		}
 	}
+
+	if !streamErrors && len(errs) != 0 {
+		w.Header().Set("X-Stream-Error", ipfsrepo.NewMultiError(errs...).Error())
```

Comment:
We should not require all go-ipfs module to re-use a tiny method. Cluster should not depend on go-ipfs.